### DISCUSSION TITLE
Cascader Component

### DISCUSSION THREAD

**Discussion by wang2lang** [mantinedev/mantine#1050] [Feature requests]
### Usage example, including component/hook API

https://github.com/react-component/cascader

**Comment by rtivital** [comment]
I haven't really seen this kind of UI in any of applications, is it common in China?

**Comment by wang2lang** [comment]
> I haven't really seen this kind of UI in any of applications, is it common in China?

YEP!
![截屏2022-01-27 下午9 43 15](https://user-images.githubusercontent.com/4401831/151370791-151cc39b-1e9f-4b4e-87ff-9fd93b4d4b42.png)

**Comment by rtivital** [comment]
Thanks, I'll consider adding it then

**Comment by lightsound** [comment]
Cascader is sometimes used in Japan.
I would be happy if it added.

**Comment by tsar-boomba** [comment]
I'd be interested in developing this component if no one else is working on it. Just to be sure, I'd create this in the mantine-labs package.

**Comment by rtivital** [comment]
go on @tsar-boomba, if you need help reach out in contributing channel on Discord

**Comment by JammyLin** [comment]
Will the component be re-opened for development? I saw tsar-boomba already finished it, and wait for v6. Now it's v6.

**Reply by tsar-boomba** [reply]
I don't currently have time to develop this, if there is still interest in early may, when my school and robotics has ended, I'd be glad to get it completed.

**Reply by JammyLin** [reply]
OK. Just do your own thing. Thank you for taking the time to do this.

**Reply by langziyang** [reply]
in China,Cascader or tree select is very common,In fact, I think there are people around the world who need. for example：category select, Most categories have children,children have children too.....,so ,when i need choose some children category, the ,Cascader or tree select is look very friendly。hope add it
